tests/topotests/isis_topo2/r1/isisd.conf

```diff
@@ -0,0 +1,19 @@
+hostname r1
```

Comment:
Why not reuse the isis_topo2 test?
If the test is new, the name should be isis_advert_high_metric instead

Reply:
Updated